airflow/jobs/scheduler_job.py

```diff
@@ -613,11 +613,6 @@ def process_file(
         3. For each DAG, see what tasks should run and create appropriate task
         instances in the DB.
         4. Record any errors importing the file into ORM
-        5. Kill (in ORM) any task instances belonging to the DAGs that haven't
-        issued a heartbeat in a while.
-
-        Returns a list of serialized_dag dicts that represent the DAGs found in
-        the file
 
```

Comment:
- `DagFileProcessor.process_file()` doesn't take care of killing zombie anymore.
- The statement of what's returned here is stale